pom.xml

```diff
@@ -909,6 +958,18 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
A primitive test to spin up a Spark shell in client mode worked. Cluster mode running SparkPi also worked. Hadoop cluster was just on my local machine running 2.5.0-CDH5.3.3. I built Spark against Hadoop 2.6.